index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="heading">
+        <a class="logo" href="#"> 
+            John Doe
+        </a>
+        <a href="#">
+            <img class="menu" src="./assets/menu_white_24dp.svg" alt="menu_bar">
+        </a>
+    </header>
+    <div class="job-title">
+        <div class="person-name">
+            <h1> Aku</h1>
+            <h2>Software Developer</h2>
+        </div>
+        <img class="ellipse" src="./assets/Ellipse18.png" alt="">
+
+    </div>
+    <p class="discription">
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hasitate to contact me.
+    </p>
```

Comment:
- [ ] Add the description class paragraph inside to the div class of job-title.

Reply:
Thank you.